specs/recipient_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'minitest'
+require_relative 'test_helper.rb'
+
+describe "Recipient" do
+  before do
+
+  end
+
+  describe "self.get" do
+    
+  end
+
+  describe "details" do
+
+  end
+
+  describe "self.list" do
+    
+  end
+end
```

Comment:
I'd also like to see a test that verifies that `self.load` `raise`s `NotImplementedError`.